kaldi/install.bash

```diff
@@ -64,18 +65,18 @@ fi
 tue-install-git "$KALDI_REPO" "$KALDI_HOME" "$KALDI_REPO_BRANCH"
 
 # Build toolkit if needed
-cd "$KALDI_HOME"
+cd "$KALDI_HOME" || tue-install-error "Missing directory: $KALDI_HOME"
```

Comment:
Similarly here. I would prefer having this check in tue-install-git as its job is to ensure cloning happens and the directory exists